_docs/2.0.0/usage/menas-api.md

```diff
@@ -0,0 +1,140 @@
+title: Usage - Menas API
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+- [Export Entity](#export-entity)
+  - [Example Import Response for Schema](#example-import-response-for-schema)
+- [Entity Import](#entity-import)
+  - [JSON payload](#json-payload)
+  - [Example Export Response for Schema](#example-export-response-for-schema)
+
+
+### Export Entity
+
+This feature exports a single entity from Menas. All exports are version less. Except for connected entities. They are kept as is, but need to be validated before import, that they match.
+
+`GET /menas/api/{entity}/exportItem/{name}/{version}`
+
+- entity is `schema`, `dataset` or `mappingTable`
+- name is the name of the entity as seen in Menas
+- version is the version of the entity as seen in Menas. Version is optional and in that case the latest version will be used
+
+#### Example Import Response for Schema
+
+**Top level keys**
+
+| Key | Value | Description |
+|---|---|---|
+| metadata | Struct | Metadata for the export functionality |
+| item | Struct | exported entity |
+
+**Metadata keys**
+
+| Key | Value | Description |
+|---|---|---|
+| exportVersion | Integer | Version of the export and model used |
+
+**Item keys**
+
+| Key | Value | Description |
+|---|---|---|
+| name | String | the name of the entity |
+| description | String | the description of the entity |
+| fields | Struct | fields specified in the schema. A complex type representing columns. |
+
+**Fields keys**
+
+| Key | Value | Description |
+|---|---|---|
+| name | String | the name of the column |
+| type | String | the type of the column |
+| path | String | parent column name |
+| absolutePath | String | absolute path to the column |
+| elementType | String | if field is a struct or array |
+| nullable | Boolean | boolean value specifying nullability |
+| metadata | Struct | metadata of the field, like patters, default value, etc. |
+| children | Array of Fields | array of fields. Sub-columns/fields to this one. |
+
+
+Example:
+```json
+{
+  "metadata": {
+    "exportVersion": 1
+  },
+  "item": {
+    "name": "Name",
+    "description": "",
+    "fields": [
+      {
+        "name": "Boolean",
+        "type": "boolean",
+        "path": "",
+        "elementType": null,
+        "containsNull": null,
+        "nullable": true,
+        "metadata": {
+          "default": null
+        },
+        "children": [],
+        "absolutePath": "Boolean"
+      },
+      {
+        "name": "Byte",
+        "type": "byte",
+        "path": "",
+        "elementType": null,
+        "containsNull": null,
+        "nullable": true,
+        "metadata": {},
+        "children": [],
+        "absolutePath": "Byte"
+      }
+    ]
+  }
+}
+```
+
+### Entity Import
+
+This feature imports a single entity. All imports are version less. If import does not find an entity with the same name it will create a new one and start the version from 1. If import finds a version it will update the previous version.
```

Comment:
```suggestion
This feature imports a single entity. All imports are versionless. If the import does not find an entity with the same name it will create a new one and start the version from 1. If the import finds an existing version, it will update the previous version.
```